go/libkb/checkers.go

```diff
@@ -14,7 +14,7 @@ import (
 
 var emailRE = regexp.MustCompile(`^\S+@\S+\.\S+$`)
 
-var deviceRE = regexp.MustCompile(`^[a-zA-Z0-9][ _'a-zA-Z0-9+-—–‘’]*$`)
+var deviceRE = regexp.MustCompile(`^[a-zA-Z0-9][ _'a-zA-Z0-9+\-—–‘’]*$`)
```

Comment:
usually i workaround this issue by just putting `-` last.  That might actually be easier to read too, given the`-` and the em-dash smush together .....

Reply:
On the other hand the escaping helps differentiate it from the en dash, although I agree that we should probably reorder them for consistency.